common/types/providers_test.go

```diff
@@ -167,6 +170,56 @@ func TestNativeToValue(t *testing.T) {
 	// Proto conversion test.
 	parsedExpr := &expr.ParsedExpr{}
 	expectNativeToValue(t, parsedExpr, NewObject(parsedExpr))
+
```

Comment:
Maybe we should make the JSON and Any tests their own functions.

Reply:
done.